src/test/java/io/stargate/sgv2/jsonapi/api/v1/UpdateManyIntegrationTest.java

```diff
@@ -0,0 +1,349 @@
+
+@QuarkusIntegrationTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class UpdateManyIntegrationTest extends CollectionResourceBaseIntegrationTest {
+  @Nested
+  @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+  class UpdateMany {
+
+    private void insert(int countOfDocument) {
+      for (int i = 1; i <= countOfDocument; i++) {
+        String json =
+            """
+                                    {
+                                      "insertOne": {
+                                        "document": {
+                                          "_id": "doc%s",
+                                          "username": "user%s",
+                                          "active_user" : true
+                                        }
+                                      }
+                                    }
+                                    """;
+        given()
+            .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+            .contentType(ContentType.JSON)
+            .body(json.formatted(i, i))
+            .when()
+            .post(CollectionResource.BASE_PATH, keyspaceId.asInternal(), collectionName)
+            .then()
+            .statusCode(200);
+      }
+    }
+
+    @Test
+    @Order(2)
+    public void updateManyById() {
+      insert(1);
+      String json =
+          """
+                                {
+                                  "updateMany": {
+                                    "filter" : {"_id" : "doc1"},
+                                    "update" : {"$set" : {"active_user": false}}
+                                  }
+                                }
+                                """;
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(json)
+          .when()
+          .post(CollectionResource.BASE_PATH, keyspaceId.asInternal(), collectionName)
+          .then()
+          .statusCode(200)
+          .body("status.matchedCount", is(1))
+          .body("status.modifiedCount", is(1))
+          .body("status.moreData", nullValue());
+
+      String expected = "{\"_id\":\"doc1\", \"username\":\"user1\", \"active_user\":false}";
+      json =
+          """
+                        {
+                          "find": {
+                            "filter" : {"_id" : "doc1"}
+                          }
+                        }
+                        """;
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(json)
+          .when()
+          .post(CollectionResource.BASE_PATH, keyspaceId.asInternal(), collectionName)
+          .then()
+          .statusCode(200)
+          .body("data.docs[0]", jsonEquals(expected));
+
+      cleanUpData();
```

Comment:
Instead of relying every test to clean up after themselves, would it make sense to call this before each test?
Otherwise failing tests could leave garbage for other test methods?

Reply:
Make sense, Will add @AfterEach to the cleanUp(), so the test doesn't need to call this.